src/buildCommon.js

```diff
@@ -636,10 +637,10 @@ const makeVList = function(params: VListParam, options: Options): DomSpan {
             }
 
             realChildren.push(childWrap);
-            currPos += elem.height + elem.depth;
+            currPos += utils.round(elem.height + elem.depth);
```

Comment:
```suggestion
            currPos += elem.height + elem.depth;
```
It seems `currPos` is not used directly and rounding is not necessary.